sample/build.gradle

```diff
@@ -28,9 +28,7 @@ apply from: '../static-analysis.gradle'
 apply from: '../common-android.gradle'
 
 android {
-    defaultConfig {
-        applicationId "com.github.venom.sample"
```

Comment:
it seems like applicationId is still recommended approach for defining app id

```
Explicitly define the application ID. If the application ID is not explicitly defined using the applicationId property, it automatically takes on the same value as the namespace.
This means that changing the namespace changes the application ID, which is usually undesired.
```

https://developer.android.com/studio/build/configure-app-module#set-application-id

Reply:
In this case it is what is expected, the applicationId will get the value from the `namespace`.
Do you want me to keep it?

Reply:
> the applicationId will get the value from the namespace.

yeah, this is my concern since changing the namespace will automatically modify the application id which is not what we always want. I'd keep them separate